woocommerce/woocommerce-ios
# [Woo POS]Revert product stock constraint

Closes #12841 

## Description
This PR reverts the initial implementation for stock/quantities management. As shared here p91TBi-bcW-p2#comment-12191 , the merchant can add products to the checkout process without taking into account the product stock.

## Testing instructions
In POS, observe that the dummy products do not show inventory levels or "Out of stock" text, and that the card colour does not change when we add products to the cart.

## Screenshots
| Before | After |
|--------|--------|
| <img width="371" alt="Screenshot 2024-05-28 at 13 43 34" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/825a3b92-97ef-437a-9764-c61955ba02e2"> | <img width="371" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/dfa5cfd4-5e46-4bea-ab4b-12d51c63638b"> |
